app/assets/stylesheets/application.scss

```diff
@@ -10,6 +10,10 @@
  * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
  * file per style scope.
  *
- *= require_tree .
- *= require_self
+ * @import
  */
+
+// "bootstrap-sprockets" must be imported before "bootstrap" and "bootstrap/variables"
+@import "bootstrap-sprockets";
+@import "bootstrap";
+@import "main";
```

Comment:
can you get rid of these new line thingys please? Sublime is known to do this. Also it's worth setting the tabs to 2 spaces.